### ISSUE TITLE
Kernel version checks on CentOS

### ISSUE THREAD

**Issue by jwbensley** [jwbensley/Etherate#36]
https://github.com/jwbensley/Etherate/blob/master/defaults.c#L228

**Comment by jwbensley** [comment]
Replace this:

```
    #if LINUX_VERSION_CODE >= KERNEL_VERSION(3,14,0)
            
        static const int32_t sock_qdisc_bypass = 1;
        int32_t sock_qdisc_ret = setsockopt(test_interface->sock_fd, SOL_PACKET, PACKET_QDISC_BYPASS, &sock_qdisc_bypass, sizeof(sock_qdisc_bypass));

        if (sock_qdisc_ret == -1) {
            perror("Error enabling QDISC bypass on socket ");
            return EXIT_FAILURE;
        }

    #endif
```


With this:
```
    #if defined(PACKET_QDISC_BYPASS)
        static const int32_t sock_qdisc_bypass = 1;
        int32_t sock_qdisc_ret = setsockopt(test_interface->sock_fd, SOL_PACKET, PACKET_QDISC_BYPASS, &sock_qdisc_bypass, sizeof(sock_qdisc_bypass));

        if (sock_qdisc_ret == -1) {
            perror("Error enabling QDISC bypass on socket ");
            return EXIT_FAILURE;
        }
    #endif
```

**Comment by jwbensley** [comment]
Remove:
`#include <linux/version.h>`

**Comment by jwbensley** [comment]
This is fixed in version 1.17 (2018-04).
